databuilder/query_model.py

```diff
@@ -221,7 +221,7 @@ def aggregate(self, function, column):
 
 @dataclass(frozen=True)
 class Table(BaseTable):
-    name: str
+    contract: None  # Because of circular imports, I can't put the correct type here
```

Comment:
You can mention types by name, as strings (if you have `from __future__ import annotations`), although I haven't checked to see if that helps with circular dependencies.